src/app/tabs/compile-tab.js

```diff
@@ -168,12 +168,8 @@ class CompileTab extends ViewPlugin {
       }
     })
 
-    globalRegistry.get('themeModule').api.events.on('themeChanged', (theme) => {
-      const invert = theme.quality === 'dark' ? 1 : 0
-      const img = document.getElementById('swarmLogo')
-      if (img) {
-        img.style.filter = `invert(${invert})`
-      }
+    globalRegistry.get('themeModule').api.events.on('themeChanged', () => {
+      globalRegistry.get('themeModule').api.fixInvert(document.getElementById('swarmLogo'))
```

Comment:
doesn't need that for the ipfs logo?

Reply:
I thought about it but it has color.. not sure. do you want me to add?

Reply:
> do you want me to add?

not necessarily, just wanted to make sure you were aware of that